docs/content/Theming/06_browser_color.md

```diff
@@ -0,0 +1,46 @@
+@ngdoc content
+@name Browser Colors
+@description
+## This feature is **mobile** only
```

Comment:
I would move this below the image and make it a `<div class="layout_note">` instead of a heading.